### PR TITLE
Set package version in Docker images + update Lerna to 6.0.3

### PR DESCRIPTION
**Pull request by tovbinm**
## Description

Package version wasn't set correctly for published Docker images. This change fixes it + updates Lerna to 6.0.3

## Type of change
- [x] Bug fix
- [ ] New feature
- [ ] Breaking change

### CODE REVIEW

**Review by cjwooo — CHANGES_REQUESTED** [review]
Currently doesn't work with source acceptance tests and makes unnecessary changes to package-lock.json. I'll take ownership of this PR.

File: Dockerfile [hunk 1]
```diff
@@ -8,9 +8,18 @@ COPY ./faros-airbyte-cdk ./faros-airbyte-cdk
...
+
 RUN apk add --no-cache --virtual .gyp python3 make g++ \
     && npm install -g lerna tsc
 RUN lerna bootstrap --hoist
+
+ARG version
+RUN test -n "$version" || (echo "'version' argument is not set, e.g --build-arg version=x.y.z" && false)
+ENV CONNECTOR_VERSION $version
+
+RUN cp package-lock.json .package-lock.json.tmp \
```
Comment (cjwooo) [inline Dockerfile:20]: Couldn't find a lerna command option that maintains the package-lock formatting. I confirmed that the corrected version still applies.

**Review by cjwooo — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (tovbinm) [reply Dockerfile:20]: Does the `package-lock.json` contain any references to local packaged, e.g `faros-airbyte-cdk`? If so, they would keep pointing to `0.0.1`, no?

Comment (cjwooo) [reply Dockerfile:20]: There's no mention of any package that is implemented in the repo.

**Pull request merged**